core/arch/arm/kernel/boot.c

```diff
@@ -1423,6 +1423,33 @@ static void init_primary(unsigned long pageable_part, unsigned long nsec_entry)
 	init_sec_mon(nsec_entry);
 }
 
+#ifdef CFG_SECURE_PARTITION
+static void check_crc32_feature(void)
+{
+	static bool checked;
```

Comment:
I removed the initialisation for these variables to get rid of the checkpatch warning

Reply:
caller could pass a argument to state whether it is the primary core that calls it (from `boot_init_primary_late()` or like) or it is a secondary core (`from init_secondary_helper()`).